bin/internal/common.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+OPTIONS="-f integration-tests/docker-compose.yml -f docker-compose.yml --project-name $(basename $(pwd))"
+
+checkIntegrationTestsDirectoryExists () {
+  if [[ ! -e 'integration-tests' ]]
+  then
+    echo "Integration tests not available, have you linked the directory? Check the README file for further details.\n"
```

Comment:
echo without `-e` doesn't interpret new lines with \n
```
 timja at potato in ~/projects/moj-reform/cmc/citizen-frontend on feature/ROC-2385-update-links*
$ echo -e "hi\n"
hi

 timja at potato in ~/projects/moj-reform/cmc/citizen-frontend on feature/ROC-2385-update-links*
$ echo  "hi\n"
hi\n
```

Reply:
That is funny because both result in the same output on my machine:

```
 ~/Pr/M/R/CM/se/bin   fix/make-tests-run-again  echo "hi\n"
hi
```

Reply:
Done it for those who uses echo that requires extra switch to enable interpretation of backslash escapes

Reply:
sh-4.2$ echo "fasds\n"
fasds\n

Reply:
at a guess it's your zsh?

in my machine, in docker and on the servers all of them require the switch 👅